src/service/lootDrop.ts

```diff
@@ -204,8 +204,22 @@ export async function postLootDrop(
                           url: "attachment://opened.gif",
                       }
                     : undefined,
+                fields: [
+                    {
+                        name: "🎉 Geschenköffner",
+                        value: winner.toString(),
+                    },
+                    ...(initialAttribute
+                        ? [
+                              {
+                                  name: "⭐ Rarität",
+                                  value: initialAttribute?.displayName,
```

Comment:
⭐ ist schon das Szmbol für "Selten". Ich würd eher einfach nur die rarität anzeigen, mit Symbol davor.
```suggestion
                                  name: `${initialAttribute.shortDisplay} ${initialAttribute.displayName}`,
                                  value: "",
```

Reply:
Geht ein empty string als Wert? Ich glaube nicht..

Reply:
Empty string geht nicht. Sieht jetzt so aus
![image](https://github.com/user-attachments/assets/5ff9e6ee-2e6a-4fff-bf3d-0057fd7f93d0)